src/modules/notification/infrastructure/store/modules/notification.ts

```diff
@@ -0,0 +1,36 @@
+import EventNotification from '@/modules/notification/domain/entity/EventNotification';
```

Comment:
```suggestion
import EventNotification from '@/modules/notification/domain/entity/EventNotification';
```

Using a resolve alias like this only works in this application alone. If you integrate it into another application (for example, our old Calendar application, with Webpack), it will break.